docs/posts/blog_DeaMaria/index.qmd

```diff
@@ -14,40 +14,40 @@ I took several online specializations and courses after finding myself without a
 Contributing to open source turned out to be the solution to the vicious circle of not being hired for a lack of work experience and not getting experience because nobody hired me.
 
 ## Knowledge about Open Source
-I started joining a Paris pyData meetup, and there I found out about an “Impact Scholarship”. This is a program sponsored by NumFOCUS. I was a lucky recipient of the scholarship which allowed me to attend for free the pyData Global 2021 online conference. I volunteered as a host for some presentations, and I met many other volunteers. Experienced participants mentioned that Open Source was great for practicing coding.
+I started joining a Paris PyData meetup, and there I found out about an “Impact Scholarship”. This is a program sponsored by NumFOCUS. I was a lucky recipient of the scholarship which allowed me to attend for free the PyData Global 2021 online conference. I volunteered as a host for some presentations, and I met many other volunteers. Experienced participants mentioned that open source was great for practicing coding.
 The program also connected me to a mentor. She was a freelancer, which is what I wanted to become. She shared with me some insights about the freelance business model. We are still in contact and I am very glad about it. 
 The following year I was asked if I could volunteer to review the talk proposals for the PyData conference as well as a Julia conference. This was very interesting.
 
 ## Finding the right project
 Afterward, I searched for OS projects on GitHub to contribute. But it hadn’t occurred to me that I could be contributing to a well-known library. I thought I would have needed to have experience in smaller projects with a smaller code base.
 
 ## Joining sprints
-I kept working alone. I built a few things to help out family members with my skills. Until there was a workshop on contributing to Scikit-learn organized by pyLadies Paris and conducted by the core devs. It was 4-sessions long. I was able to master the contribution workflow and contribute a couple of Pull Requests.
+I kept working alone. I built a few things to help out family members with my skills. Until there was a workshop on contributing to scikit-learn organized by PyLadies Paris and conducted by the core devs. It was 4 sessions long. I was able to master the contribution workflow and contribute a couple of pull requests.
```

Comment:
scikit-learn's website is not 100% consistent, but mostly it uses no capitalization for scikit, so I'm wondering if there's a correct capitalization? Either way is fine with me, this is a very tiny nitpick.

Reply:
@melissawm Thank you for the review! In my observation, the first letter in the name of the project is capitalized only at the start of the sentence which is the standing rule in written English (and most modern languages).